lib/jquery-rjs/rendering.rb

```diff
@@ -8,6 +8,7 @@ def render_with_update(options = {}, locals = {}, &block)
       render_without_update(options, locals, &block)
     end
   end
-  
-  alias_method_chain :render, :update
-end
\ No newline at end of file
+
+  alias_method :render_without_update, :render
```

Comment:
Should these use prepend?

Reply:
@bdunne seem my commit message for this:

https://github.com/ManageIQ/jquery-rjs/pull/2/commits/417da52ad67928a9be61b16073cc663984c6f225

>  **[rails6] replace alias_method_chain**
> 
> While using the `Module.prepend` approach is probably the correct way to
> do this, the way that this class is defined is pretty gross, so I would
> rather avoid trying to deal with that and simply replicate the
> functionality `alias_method_chain` provided by doing two `alias_method`
> calls.
> 
> This is a Gem the probably should just die anyway, so I am not terribly
> bothered by the hack...

Reply:
Also, since this is mostly just to maintain legacy code, I am trying to do the least amount of changes possible, and using `prepend` is doing more refactoring than I would like, where this approach is basically taking what `alias_method_chain` did and doing it manually.